.travis.yml

```diff
@@ -28,97 +53,38 @@ matrix:
                   notification_email: menoh-oss@preferred.jp
                   build_command_prepend: >-
                       cov-configure --compiler /usr/bin/g++-7 --comptype g++ -- -march=native -fPIC -std=gnu++14 &&
-                      cmake .
+                          cmake
+                              -DMKLDNN_INCLUDE_DIR="$HOME/mkl-dnn-${MKLDNN_VERSION}/include"
+                              -DMKLDNN_LIBRARY="$HOME/mkl-dnn-${MKLDNN_VERSION}/lib/libmkldnn.so" .
```

Comment:
It failed with the following error:
```
Usage
  cmake [options] <path-to-source>
  cmake [options] <path-to-existing-build>
Specify a source directory to (re-)generate a build system for it in the
current working directory.  Specify an existing build directory to
re-generate its build system.
Run 'cmake --help' for more information.
bash: line 76: -DMKLDNN_INCLUDE_DIR=/home/travis/mkl-dnn-0.16/include: No such file or directory
```

This is because the `build_command_prepend` value is interpreted as `"cov-configure --compiler /usr/bin/g++-7 --comptype g++ -- -march=native -fPIC -std=gnu++14 &&\n    cmake\n        -DMKLDNN_INCLUDE_DIR=\"$HOME/mkl-dnn-${MKLDNN_VERSION}/include\"\n        -DMKLDNN_LIBRARY=\"$HOME/mkl-dnn-${MKLDNN_VERSION}/lib/libmkldnn.so\" ."`.

It seems that we need to write as follows (same indentation across lines):
```
                  build_command_prepend: >-
                      cov-configure --compiler /usr/bin/g++-7 --comptype g++ -- -march=native -fPIC -std=gnu++14 &&
                      cmake
                      -DMKLDNN_INCLUDE_DIR="$HOME/mkl-dnn-${MKLDNN_VERSION}/include"
                      -DMKLDNN_LIBRARY="$HOME/mkl-dnn-${MKLDNN_VERSION}/lib/libmkldnn.so" .
```

Reply:
Good catch. Fixed.